src/rrdp.rs

```diff
@@ -390,25 +390,28 @@ impl RrdpState {
     /// temporary file and then rename it to avoid serving partially
     /// written files.
     pub fn write_notification(&self) -> Result<()> {
-        let tmp_path = self.rrdp_dir.join("notification.tmp");
-        let final_path = self.rrdp_dir.join("notification.xml");
+        let notification_file_filename_final = self.notification_uri.notify_filename()?;
+        let notification_file_filename_tmp = format!("{}.tmp", notification_file_filename_final);
 
-        info!("Updating notification file at {:?}", final_path);
+        let path_final = self.rrdp_dir.join(notification_file_filename_final);
```

Comment:
Is this filename guaranteed to be a valid file name?

Reply:
indeed and I don't see how it could be invalid under normal circumstances.. the temp file will be normal filename with a '.tmp' postfix. But.. it's just a name - I assume noone is using fat32 or something and a name like 'notifcation.xml.tmp' is just fine..